File: server/ITI-graduation-project-Advilaw-main/AdviLaw.Domain/Entites/JobSection/JobStatus.cs
namespace AdviLaw.Domain.Entites.JobSection
{
    public enum JobStatus
    {
        WaitingApproval = 0,
        NotAssigned = 1,
        WaitingAppointment=2,
        WaitingPayment=3,
        NotStarted=4,
        LawyerRequestedAppointment=5,
        ClientRequestedAppointment=6,
        Accepted ,
        Rejected ,
        Started,
        Ended,
    }
}

File: client/ITI-graduation-project-Advilaw-Frontend-main/src/app/types/Jobs/JobStatus.ts
export enum JobStatus {
  WaitingApproval = 'WaitingApproval',
  NotAssigned = 'NotAssigned',
  WaitingAppointment = 'WaitingAppointment',
  WaitingPayment = 'WaitingPayment',
  NotStarted = 'NotStarted',
  LawyerRequestedAppointment = 'LawyerRequestedAppointment',
  ClientRequestedAppointment = 'ClientRequestedAppointment',
  Accepted = 'Accepted',
  Rejected = 'Rejected',
  Started = 'Started',
  Ended = 'Ended',
}
